src/codeu/chat/client/commandline/Chat.java

```diff
@@ -517,7 +628,8 @@ public void invoke(List<String> args) {
         System.out.println("Conversation Info:");
         System.out.format("  Title : %s\n", conversation.conversation.title);
         System.out.format("  Id    : UUID:%s\n", conversation.conversation.id);
-        System.out.format("  Owner : %s\n", conversation.conversation.owner);
+        System.out.format("  Creator  : %s\n", conversation.conversation.owner);
+        System.out.format("  Access Level: %s\n", conversation.getUserAccessLevel().getAccessLevel());
```

Comment:
... except now I see that it's not the default access level, it's this conversation's access level for the current user.

In that case, my suggestion changes to "add another line to print out the default access level".